external/model-preparation-algorithm/mpa_tasks/apis/detection/task.py

```diff
@@ -177,7 +177,15 @@ def _init_recipe(self):
             # recipe = os.path.join(recipe_root, 'pretrain.yaml')
             raise NotImplementedError(f'train type {train_type} is not implemented yet.')
         elif train_type == TrainType.Incremental:
-            recipe = os.path.join(recipe_root, 'imbalance.py')
+            if self._model_name.split('-')[0] == 'YOLOX':
```

Comment:
I also left a comment related to the PR of the MPA side. 
Instead of separating the recipe according to the model, I think you can process ignore=false and put the data_pipeline directly into the OTE model config folder if necessary. 

For an example, refer to instance-segmentation.